src/app/components/RecipeCards/CardDefault/index.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  Box,
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  ListItem,
+  Typography,
+} from '@mui/material';
+import React from 'react';
+
+export type StepItemType = {
+  id: number;
+  title: string;
+  description: string;
+};
+
+type Props = {
+  item: StepItemType;
+};
+
+export default (props: Props): JSX.Element => {
```

Comment:
you could destructure props here and maybe items down below so you dont have to write props.item.name etc all the time

Reply:
you're right, it would look cleaner.
I think I'm going to stick to this form for now, and destructure all props later when I have time to clean the code. :)